### DISCUSSION TITLE
WPF中 MenuItem怎么做到VS的展开效果

### DISCUSSION THREAD

**Discussion by AIKA1024** [BYJRK/DotNet-Discussions#90] [WPF 开发]
我的MenuItem的边框是通过BorderThickness和Effect画出来的，但是MenuItem和Popup之间我想去掉一部分的线
**VS的**
![image](https://user-images.githubusercontent.com/68951745/236662127-f3a91009-f062-4a6c-b7e3-6bebd198c550.png)
**我实现的**
![image](https://user-images.githubusercontent.com/68951745/236662141-bdb312c5-9d87-45f0-bbed-1dbdd6d2d8d4.png)
这该怎么做

**Comment by CodingOctocat** [comment]
这得重写个 MenuItem 吧，得不偿失，感觉挺麻烦的。
